docs/references/events.md

```diff
@@ -50,7 +50,7 @@ This indicates that the Workflow Execution has successfully completed. The Event
 | -------------------------------- | ----------------------------------------------------------------------------------------------- |
 | result                           | Serialized result of completed Workflow.                                                        |
 | workflow_task_completed_event_id | The Id of the [WorkflowTaskCompleted](#workflowtaskcompleted) that the Event was reported with. |
-| new_execution_run_id             | The [Run Id](/concepts/what-is-a-run-id) of the new Workflow started by cron.                   |
+| new_execution_run_id             | The [Run Id](/concepts/what-is-a-run-id) of the new Workflow started by Cron.                   |
```

Comment:
Despite the terms Cron Job and Cron Schedule, when we refer to cron by itself, the styling is lowercase—I think. @flossypurse, concur or disagree?

Reply:
Sure, but I don't know of when we would use it generically - 
It is either Temporal Cron Job, or Cron Schedule I think.